src/domain/synthetics/markets/useMarketsInfoRequest/buildMarketsConfigsRequest.ts

```diff
@@ -220,6 +220,30 @@ export async function buildMarketsConfigsRequest(
           methodName: "getBytes32",
           params: [prebuiltHashedKeys.virtualLongTokenId],
         },
+        optimalUsageFactorLong: {
```

Comment:
if we need these values only on stats page, let's move it to separated request please, we already have too many of calls for markets, plan to separate it to several requests in the future

Reply:
Good suggestion! Moved this keys to a separate prebuilt file and separate request.